### ISSUE TITLE
Can this library run under Linux?

### ISSUE THREAD

**Issue by 56756639** [minervous/QtStreamDeck#11]
I want to develop an application recently, and I hope this application can make my Stream Deck MK.2 work properly on Ubuntu system. Can this library help me?

**Comment by GooRoo** [comment]
Never tested it on Linux, but I don't see any reason why this wouldn't work. Are you going to use C++?

**Comment by 56756639** [comment]
Yes, I plan to use C++ to implement my project. Although I would prefer to use C, I can't really find a C library for Stream Deck

**Comment by GooRoo** [comment]
This library focuses mostly on Qt and is heavily inspired by https://github.com/abcminiuser/python-elgato-streamdeck. The latter one is not that complicated. You could use it as a reference to make your own C implementation based on libusb and hidapi.

**Comment by 56756639** [comment]
My idea is to use Qt to achieve similar effects as the official Stream Deck software, and then port it to Linux for running (by cross-compiling to generate an executable file for the Linux platform).I don't know if my idea can be realized, but I want to try.
